src/libsync/configfile.cpp

```diff
@@ -121,10 +121,10 @@ bool ConfigFile::setConfDir(const QString &value)
     return false;
 }
 
-bool ConfigFile::optionalDesktopNotifications() const
+bool ConfigFile::optionalServerNotifications() const
 {
     QSettings settings(configFile(), QSettings::IniFormat);
-    return settings.value(QLatin1String(optionalDesktopNoficationsC), true).toBool();
+    return settings.value(QLatin1String(optionalServerNotificationsC), false).toBool();
```

Comment:
I think notifications should be enabled by default. Any opinions @jancborchardt

Reply:
Done.